2025-06-23 05:28:16,591 INFO: Auto-recovery completed successfully [in /home/sinnys/NIDPS/nidps/core/recovery.py:180]
2025-06-23 05:28:49,180 WARNING: ALERT: Rule 'Potential TCP Port Scan (SYN)' (MEDIUM) matched: Ether / IP / TCP 192.168.0.149:35488 > 185.125.190.96:http S [in /home/sinnys/NIDPS/nidps/core/engine.py:88]
2025-06-23 05:29:04,962 INFO: Advanced analytics stopped [in /home/sinnys/NIDPS/nidps/core/analytics.py:76]
2025-06-23 05:29:04,962 INFO: Auto-recovery monitoring stopped [in /home/sinnys/NIDPS/nidps/core/recovery.py:69]
2025-06-23 05:29:07,521 INFO: WebSocket real-time updates stopped [in /home/sinnys/NIDPS/nidps/core/websocket_manager.py:77]
2025-06-23 05:29:07,522 INFO: NIDPS Engine Stopped. [in /home/sinnys/NIDPS/nidps/core/engine.py:221]
2025-06-23 05:29:10,260 INFO: Attempting to start sniffer on interface default [in /home/sinnys/NIDPS/nidps/core/sniffer.py:16]
2025-06-23 05:29:10,262 INFO: Advanced analytics started [in /home/sinnys/NIDPS/nidps/core/analytics.py:69]
2025-06-23 05:29:10,264 INFO: Auto-recovery monitoring started [in /home/sinnys/NIDPS/nidps/core/recovery.py:62]
2025-06-23 05:29:10,269 INFO: WebSocket real-time updates started [in /home/sinnys/NIDPS/nidps/core/websocket_manager.py:70]
2025-06-23 05:29:10,275 INFO: NIDPS Engine Started with all advanced features. [in /home/sinnys/NIDPS/nidps/core/engine.py:193]
2025-06-23 05:31:27,801 ERROR: Error in packet inspection: 'signatures' [in /home/sinnys/NIDPS/nidps/core/dpi.py:114]
2025-06-23 05:31:50,510 INFO: Advanced analytics stopped [in /home/sinnys/NIDPS/nidps/core/analytics.py:76]
2025-06-23 05:31:55,511 INFO: Auto-recovery monitoring stopped [in /home/sinnys/NIDPS/nidps/core/recovery.py:69]
2025-06-23 05:32:00,275 INFO: WebSocket real-time updates stopped [in /home/sinnys/NIDPS/nidps/core/websocket_manager.py:77]
2025-06-23 05:32:00,275 INFO: NIDPS Engine Stopped. [in /home/sinnys/NIDPS/nidps/core/engine.py:221]
2025-06-23 05:34:39,295 INFO: NIDPS startup [in /home/sinnys/NIDPS/nidps/__init__.py:57]
2025-06-23 05:34:46,002 INFO: Prevention Engine initialized with 30 minute dwell time [in /home/sinnys/NIDPS/nidps/core/prevention.py:18]
2025-06-23 05:35:23,670 INFO: NIDPS startup [in /home/sinnys/NIDPS/nidps/__init__.py:57]
2025-06-23 05:35:55,454 INFO: NIDPS startup [in /home/sinnys/NIDPS/nidps/__init__.py:57]
2025-06-23 05:44:54,350 INFO: NIDPS startup [in /home/sinnys/NIDPS/nidps/__init__.py:57]
2025-06-23 05:44:57,341 INFO: Prevention Engine initialized with 30 minute dwell time [in /home/sinnys/NIDPS/nidps/core/prevention.py:18]
2025-06-23 05:45:01,534 INFO: Attempting to start sniffer on interface default [in /home/sinnys/NIDPS/nidps/core/sniffer.py:16]
2025-06-23 05:45:01,534 INFO: Advanced analytics started [in /home/sinnys/NIDPS/nidps/core/analytics.py:69]
2025-06-23 05:45:01,535 INFO: Auto-recovery monitoring started [in /home/sinnys/NIDPS/nidps/core/recovery.py:62]
2025-06-23 05:45:01,535 INFO: WebSocket real-time updates started [in /home/sinnys/NIDPS/nidps/core/websocket_manager.py:70]
2025-06-23 05:45:01,535 INFO: NIDPS Engine Started with all advanced features. [in /home/sinnys/NIDPS/nidps/core/engine.py:193]
2025-06-23 05:45:02,536 WARNING: Auto-recovery triggered due to: Engine not running [in /home/sinnys/NIDPS/nidps/core/recovery.py:159]
2025-06-23 05:45:02,537 INFO: Configuration backed up to backups [in /home/sinnys/NIDPS/nidps/core/recovery.py:262]
2025-06-23 05:45:02,537 INFO: Restarting NIDPS engine... [in /home/sinnys/NIDPS/nidps/core/recovery.py:188]
2025-06-23 05:45:07,549 INFO: Advanced analytics stopped [in /home/sinnys/NIDPS/nidps/core/analytics.py:76]
2025-06-23 05:45:07,549 ERROR: Error stopping NIDPS: cannot join current thread [in /home/sinnys/NIDPS/nidps/core/engine.py:230]
2025-06-23 05:45:09,550 INFO: NIDPS is already running. [in /home/sinnys/NIDPS/nidps/core/engine.py:179]
2025-06-23 05:45:14,550 INFO: Engine restart successful [in /home/sinnys/NIDPS/nidps/core/recovery.py:200]
2025-06-23 05:45:14,550 INFO: Restarting packet sniffer... [in /home/sinnys/NIDPS/nidps/core/recovery.py:211]
2025-06-23 05:45:15,551 INFO: Attempting to start sniffer on interface default [in /home/sinnys/NIDPS/nidps/core/sniffer.py:16]
2025-06-23 05:45:15,552 INFO: Sniffer restart successful [in /home/sinnys/NIDPS/nidps/core/recovery.py:226]
2025-06-23 05:45:15,601 INFO: Memory cleared [in /home/sinnys/NIDPS/nidps/core/recovery.py:237]
2025-06-23 05:45:15,601 INFO: Configuration backed up to backups [in /home/sinnys/NIDPS/nidps/core/recovery.py:262]
2025-06-23 05:45:15,601 INFO: Auto-recovery completed successfully [in /home/sinnys/NIDPS/nidps/core/recovery.py:180]
2025-06-23 05:46:48,084 WARNING: ALERT: Rule 'Suspicious Large Packets' (MEDIUM) matched: Ether / IP / TCP 54.84.187.34:https > 192.168.0.149:32780 A / Raw [in /home/sinnys/NIDPS/nidps/core/engine.py:88]
2025-06-23 05:48:49,179 WARNING: ALERT: Rule 'Potential TCP Port Scan (SYN)' (MEDIUM) matched: Ether / IP / TCP 192.168.0.149:40790 > 185.125.190.96:http S [in /home/sinnys/NIDPS/nidps/core/engine.py:88]
2025-06-23 05:49:13,358 WARNING: ALERT: Rule 'Potential TCP Port Scan (SYN)' (MEDIUM) matched: Ether / IP / TCP 192.168.0.149:34376 > 13.107.5.93:https S [in /home/sinnys/NIDPS/nidps/core/engine.py:88]
2025-06-23 05:49:13,461 WARNING: ALERT: Rule 'Suspicious Large Packets' (MEDIUM) matched: Ether / IP / TCP 13.107.5.93:https > 192.168.0.149:34376 A / Raw [in /home/sinnys/NIDPS/nidps/core/engine.py:88]
2025-06-23 05:49:13,461 WARNING: ALERT: Rule 'Suspicious Large Packets' (MEDIUM) matched: Ether / IP / TCP 13.107.5.93:https > 192.168.0.149:34376 A / Raw [in /home/sinnys/NIDPS/nidps/core/engine.py:88]
2025-06-23 05:49:13,478 WARNING: ALERT: Rule 'Suspicious Large Packets' (MEDIUM) matched: Ether / IP / TCP 13.107.5.93:https > 192.168.0.149:34376 PA / Raw [in /home/sinnys/NIDPS/nidps/core/engine.py:88]
2025-06-23 05:51:47,555 WARNING: ALERT: Rule 'Suspicious Large Packets' (MEDIUM) matched: Ether / IP / TCP 54.84.187.34:https > 192.168.0.149:32780 A / Raw [in /home/sinnys/NIDPS/nidps/core/engine.py:88]
2025-06-23 05:53:09,773 WARNING: ALERT: Rule 'Potential TCP Port Scan (SYN)' (MEDIUM) matched: Ether / IP / TCP 192.168.0.149:53282 > 13.107.5.93:https S [in /home/sinnys/NIDPS/nidps/core/engine.py:88]
2025-06-23 05:53:09,780 WARNING: ALERT: Rule 'Potential TCP Port Scan (SYN)' (MEDIUM) matched: Ether / IP / TCP 192.168.0.149:53298 > 13.107.5.93:https S [in /home/sinnys/NIDPS/nidps/core/engine.py:88]
2025-06-23 05:53:09,799 WARNING: ALERT: Rule 'Suspicious Large Packets' (MEDIUM) matched: Ether / IP / TCP 13.107.5.93:https > 192.168.0.149:53282 A / Raw [in /home/sinnys/NIDPS/nidps/core/engine.py:88]
2025-06-23 05:53:09,800 WARNING: ALERT: Rule 'Suspicious Large Packets' (MEDIUM) matched: Ether / IP / TCP 13.107.5.93:https > 192.168.0.149:53282 A / Raw [in /home/sinnys/NIDPS/nidps/core/engine.py:88]
2025-06-23 05:53:09,806 WARNING: ALERT: Rule 'Suspicious Large Packets' (MEDIUM) matched: Ether / IP / TCP 13.107.5.93:https > 192.168.0.149:53298 A / Raw [in /home/sinnys/NIDPS/nidps/core/engine.py:88]
2025-06-23 05:53:09,807 WARNING: ALERT: Rule 'Suspicious Large Packets' (MEDIUM) matched: Ether / IP / TCP 13.107.5.93:https > 192.168.0.149:53298 A / Raw [in /home/sinnys/NIDPS/nidps/core/engine.py:88]
2025-06-23 05:53:09,814 WARNING: ALERT: Rule 'Suspicious Large Packets' (MEDIUM) matched: Ether / IP / TCP 13.107.5.93:https > 192.168.0.149:53282 A / Raw [in /home/sinnys/NIDPS/nidps/core/engine.py:88]
2025-06-23 05:53:09,829 WARNING: ALERT: Rule 'Suspicious Large Packets' (MEDIUM) matched: Ether / IP / TCP 13.107.5.93:https > 192.168.0.149:53298 A / Raw [in /home/sinnys/NIDPS/nidps/core/engine.py:88]
2025-06-23 05:53:49,181 WARNING: ALERT: Rule 'Potential TCP Port Scan (SYN)' (MEDIUM) matched: Ether / IP / TCP 192.168.0.149:46882 > 91.189.91.49:http S [in /home/sinnys/NIDPS/nidps/core/engine.py:88]
2025-06-23 05:54:09,790 INFO: NIDPS startup [in /home/sinnys/NIDPS/nidps/__init__.py:57]
2025-06-23 05:54:13,387 INFO: Prevention Engine initialized with 30 minute dwell time [in /home/sinnys/NIDPS/nidps/core/prevention.py:18]
2025-06-23 05:56:17,984 INFO: NIDPS startup [in /home/sinnys/NIDPS/nidps/__init__.py:57]
2025-06-23 05:56:20,957 INFO: Prevention Engine initialized with 30 minute dwell time [in /home/sinnys/NIDPS/nidps/core/prevention.py:18]
2025-06-27 10:29:55,246 INFO: NIDPS startup [in /home/sinnys/NIDPS/nidps/__init__.py:57]
2025-06-27 10:30:13,001 INFO: NIDPS startup [in /home/sinnys/NIDPS/nidps/__init__.py:57]
2025-06-27 10:30:39,709 INFO: NIDPS startup [in /home/sinnys/NIDPS/nidps/__init__.py:57]
2025-06-27 19:27:11,487 INFO: NIDPS startup [in /home/sinnys/NIDPS/nidps/__init__.py:57]
2025-06-27 19:27:34,429 INFO: Prevention Engine initialized with 30 minute dwell time [in /home/sinnys/NIDPS/nidps/core/prevention.py:18]
2025-06-27 19:27:45,078 INFO: Attempting to start sniffer on interface default [in /home/sinnys/NIDPS/nidps/core/sniffer.py:16]
2025-06-27 19:27:45,078 INFO: Advanced analytics started [in /home/sinnys/NIDPS/nidps/core/analytics.py:69]
2025-06-27 19:27:45,079 INFO: Auto-recovery monitoring started [in /home/sinnys/NIDPS/nidps/core/recovery.py:62]
2025-06-27 19:27:45,079 INFO: WebSocket real-time updates started [in /home/sinnys/NIDPS/nidps/core/websocket_manager.py:70]
2025-06-27 19:27:45,079 INFO: NIDPS Engine Started with all advanced features. [in /home/sinnys/NIDPS/nidps/core/engine.py:193]
2025-06-27 19:27:46,080 WARNING: Auto-recovery triggered due to: Engine not running [in /home/sinnys/NIDPS/nidps/core/recovery.py:159]
2025-06-27 19:27:46,080 INFO: Configuration backed up to backups [in /home/sinnys/NIDPS/nidps/core/recovery.py:262]
2025-06-27 19:27:46,081 INFO: Restarting NIDPS engine... [in /home/sinnys/NIDPS/nidps/core/recovery.py:188]
2025-06-27 19:27:51,093 INFO: Advanced analytics stopped [in /home/sinnys/NIDPS/nidps/core/analytics.py:76]
2025-06-27 19:27:51,093 ERROR: Error stopping NIDPS: cannot join current thread [in /home/sinnys/NIDPS/nidps/core/engine.py:230]
2025-06-27 19:27:53,094 INFO: NIDPS is already running. [in /home/sinnys/NIDPS/nidps/core/engine.py:179]
